Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM node:lts-alpine as base
+FROM base as builder
+WORKDIR /home/node/app
+
+RUN apk update && apk add git
+
+# Install dependencies first to not invalidate the cache on every source change
+COPY package.json yarn.lock ./
+RUN yarn install --production=false --pure-lockfile
+
+# Copy stuff needed to build the frontend
+COPY locales locales
+COPY public public
+COPY webpack.*.js .babelrc ./
+
+# Build the frontend
+RUN yarn run build
+
+# Remove all devDependencies
+RUN yarn install --production=true --pure-lockfile
+
+# Production container
+FROM base
+
+# Setup various metadata
+WORKDIR /home/node/app
+USER node
+ENV NODE_ENV=production
+EXPOSE 3000
+HEALTHCHECK CMD node healthcheck.mjs
```

Comment:
Please don't enable a healthcheck by default. It usually just delays the startup of the container by 10s of seconds and the availability in proxies like Traefik.